desk

```diff
@@ -114,56 +114,95 @@ cmd_run() {
 }
 
 
+# Usage:        desk list [options]
+# Description:  List all desks along with a description
+# --only-names: List only the names of the desks
+# --no-format:  Use ' - ' to separate names from descriptions
```

Comment:
Yeah, well the reason I put the `usage` function in there was primarily for code documentation, but once I actually got around to formatting the output of the `usage` cmd (I was basing it off of the way Docker's usage help is formatted), it was painfully overladen with `\t`s and `\n`s and such. So I figured pulling it out to function descriptions would both provide good in-code documentation and allow us to automatically produce some usage help commands from the descriptions as well.